src/app/public/modules/data-manager/models/data-manager-state.ts

```diff
@@ -19,12 +19,12 @@ import {
 } from './data-view-state-options';
 
 export class SkyDataManagerState {
-  public activeSortOption: SkyDataManagerSortOption;
-  public additionalData: any;
-  public filterData: SkyDataManagerFilterData;
-  public onlyShowSelected: boolean;
-  public searchText: string;
-  public selectedIds: string[];
+  public activeSortOption?: SkyDataManagerSortOption;
```

Comment:
I wouldn't expect a class to have optional properties; if an object is of this type, it should be created through a constructor which would then add the properties to the object. This is unlike an interface where any object with the same properties will match the interface, regardless of whether it is an object literal or created through a constructor function.